app/printer.py

```diff
@@ -0,0 +1,19 @@
+from abc import ABC, abstractmethod
+
+
+class Printer(ABC):
+    @abstractmethod
+    def print_book(self, title: str, content: str) -> None:
+        pass
+
+
+class ConsolePrinter(Printer):
+    def print_book(self, title: str, content: str) -> None:
+        print(f"Printing the book: {title}...")
+        print(content)
+
+
+class ReverseConsolePrinter(Printer):
+    def print_book(self, title: str, content: str) -> None:
+        print(f"Printing the book in reverse: {title}...")
+        print(content[::-1])
```

Comment:
The same logic applies to `ReverseDisplay`. It should be renamed to `DisplayReverse`. The class name should indicate that it displays content in reverse order, with "Display" (what it does) coming first, followed by "Reverse" (how it does it).